egui/src/response.rs

```diff
@@ -277,6 +277,27 @@ impl Response {
         self.drag_released
     }
 
+    /// Mouse dragged, nevertheless it's stuck on screen's edge or not.
+    /// The idea of this method is to overcome limitations of computing delta from `old-new`,
+    /// which does not work when mouse is stuck on screen corner.
+    /// On platform where it is not supported in fallbacks to `pointer.delta()`
+    pub fn motion_delta(&self) -> Vec2 {
+        #[cfg(target_family = "wasm")]
+        return self.ctx.input().pointer.delta();
+
+        #[cfg(not(target_family = "wasm"))]
+        {
+            let res = self.ctx.input().pointer.motion();
+
+            if res == Vec2::ZERO {
+                // in a case of touchscreen or whatever
+                self.ctx.input().pointer.delta()
+            } else {
+                res
+            }
+        }
+    }
```

Comment:
It's not `egui`s responsibility to have platform-specific code like this. Instead it is up to the integrations to feed `egui` with the correct input.

In this case the winit backend should feed different data than the web backend. [Other integrations](https://github.com/emilk/egui#integrations) needs to do whats right for them.

 So if this is called `mtion_delta` then it should probably use `self.ctx.input().pointer.motion()` regardless of what the compilation target is.

Also: 'wasm" doesn't mean "web". There are egui users who run egui in wasmtime on desktop.